src/resources/nfts/simplehash/utils.ts

```diff
@@ -166,7 +169,9 @@ export function simpleHashNFTToUniqueAsset(nft: SimpleHashNFT): UniqueAsset {
     image_thumbnail_url: lowResUrl,
     image_url: imageUrl,
     isPoap: nft.contract_address.toLowerCase() === POAP_NFT_ADDRESS,
-    isSendable: nft.chain === SimpleHashChain.Ethereum,
+    isSendable:
```

Comment:
wonder if we should optimistically mark all as sendable, except for poaps since we don't support gnosis chain yet

i think our handling is pretty robust now & this way if there is an error we can at least track it via sentry